File: 프로그래머스/lv1/이하은/음양 더하기.kt
class Solution {
    fun solution(absolutes: IntArray, signs: BooleanArray): Int {
        var answer: Int = 0
        
        signs.forEachIndexed { idx, positive ->
            if(!positive){
                answer += -absolutes[idx]
            }else{
                answer += absolutes[idx]
            }
        }
        
        return answer
    }
}

File: 프로그래머스/lv1/김현진/음양 더하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> absolutes, vector<bool> signs) {
      int answer = 0;
    for(int i=0;i<signs.size();i++)
    {
        if(signs[i])//+이면
        {
            answer += absolutes[i];
        }
        else if(!signs[i])
        {
            answer -= absolutes[i];
        }
    }
  
    return answer;
}
